eslint.config.js

```diff
@@ -1,5 +1,167 @@
 import blumilkDefault from '@blumilksoftware/eslint-config'
+import stylisticPlugin from '@stylistic/eslint-plugin'
```

Comment:
`@stylistic/eslint-plugin` is plugin to `eslint` with formating rules. But `Eslint` is for linting not exactly for formatting. So maybe better idea will be use https://prettier.io/.